src/amazonica/core.clj

```diff
@@ -920,10 +932,9 @@
                          (name fname) args)))))))
 
 (defn- client-methods
-  "Returns a map with keys of idiomatic Clojure hyphenated
-   keywords corresponding to the public Java method names of
-   the class argument, vals are vectors of
-   java.lang.reflect.Methods."
+  "Returns a map with keys of idiomatic Clojure hyphenated keywords
+  corresponding to the public Java method names of the class argument, vals are
+  vectors of java.lang.reflect.Methods."
```

Comment:
Note: this didn't actually change except for the reflow. (It was different during an intermediate stage of the PR, but then I found a less intrusive way of making it work.)